src/schedule_synthesis.jl

```diff
@@ -72,16 +72,16 @@ function schedule_xghtc(constraints::Vector{<:MeetAny}; slotsize::Int64=1, H::In
 end
 
 """
-    synthesize_constraints(sysd, K, z_0, d_max, maxwindow, n, t)
+    synthesize_constraints(sysd, K, z_0, d_max, maxwindow, n, H; iterations=ceil(H/n))
 
 Find all `MeetAny` weakly hard constraints with window size at most `maxwindow` that 
 guarantees the deviation upper bound is at most `d_max`. The system is specified by 
-[`Automaton`](@ref) `a` and initial state is `z_0`. `n` and `t` are as in 
-[`bounded_runs_iter`](@ref).
+[`Automaton`](@ref) `a` and initial state is `z_0`. `n` and `iterations` are as `n` 
```

Comment:
I don't know how I feel about changing the API in this function, but not in `bounded_runs_iter`.  Overall, I like the idea of specifying the time horizon and number of steps per iteration, as it seems like those are the parameters a user would be more interested in.  But how about we separate that out into another MR and merge it after the ESWEEK deadline, to avoid breaking things unnecessarily while we're under time pressure?